packages/lesswrong/components/tagging/TagSubforumPage2.tsx

```diff
@@ -469,15 +485,26 @@ const TagSubforumPage2 = ({classes}: {
     return <div className={classNames(classes.centralColumn, classes.feedWrapper)}>
       <div className={classes.feedHeader}>
         <div className={classes.feedHeaderButtons}>
-          <SectionButton>
-            <AddBoxIcon /> New Post
-          </SectionButton>
-          <SectionButton>
+          <Link to={`/newPost?subforumTagId=${tag._id}`} className={classes.newPostLink}>
+            <SectionButton>
+              <AddBoxIcon /> New Post
+            </SectionButton>
+          </Link>
+          <SectionButton onClick={clickNewDiscussion}>
             <AddBoxIcon /> New Discussion
           </SectionButton>
         </div>
         <PostsListSortDropdown value={query.sortedBy} options={subforumSortings} />
       </div>
+      {newDiscussionOpen && <div className={classes.newDiscussionContainer}>
+        <CommentsNewForm
+          tag={tag}
+          tagCommentType={"SUBFORUM"}
+          successCallback={refetch}
+          type="comment"
+          enableGuidelines={false}
```

Comment:
I still think we should optimise for the mobile experience somewhat, and that maybe making this `"minimalist"` on mobile would be better for that